### DISCUSSION TITLE
Do you know a user with all Achievements with highest tier at the same time?

### DISCUSSION THREAD

**Discussion by parallels999** [Schweinepriester/github-profile-achievements#925] [Show and tell]
https://github.com/Schweinepriester/github-profile-achievements/blob/820678e8248410fd7dee40a506431ad4b92b2ccd/README.md?plain=1#L103

**User:** `https://github.com/messense`

![image](https://github.com/user-attachments/assets/10ff7bcd-4663-4d33-bf62-cdff0ae68921)

**CC:** @Schweinepriester

**Comment by Schweinepriester** [comment]
Nice one, thank you!

**Comment by Ankitjarwall** [comment]
i found the closest one : https://github.com/Schweinepriester

Note : Currently GitHub have only 7 achievement  out of 9 shown here, if you are searching for all 9 as shown. Search for old users and still active as they may have all of these.

Thank you.

If found my answer useful than mark it as answered.

**Comment by ThaiHoang-newbie** [comment]


**Reply by parallels999** [reply]
I don't know why you're answering, it wasn't a question, I already found one and published it
> **User:** [messense](https://github.com/messense)

**Reply by ThaiHoang-newbie** [reply]
Okay okay, I didn't see that. I will delete mine

**Comment by M-Abdullah-91** [comment]
No
